SparkPost/2web2ui
# SA-2305: Sets custom values as being lowercase

### What Changed

- Custom values for input value in typeaheads now get set to lowercase

### How To Test

- Go to report builder
- Go to filters or compare form
- Enter custom value with uppercase letters
- Verify that string changed to lowercase
- Verify that uppercase strings can't be passed in

### To Do
